tsconfig.json

```diff
@@ -9,7 +9,6 @@
     "noUnusedParameters": true,
     "noImplicitReturns": true,
     "noFallthroughCasesInSwitch": true,
-    "strictFunctionTypes": true,
```

Comment:
why?

Reply:
just to run test locally, I'll get rid of this once it's stable